src/nav-soknad/utils/rest-utils.ts

```diff
@@ -9,6 +9,27 @@ import {
 } from "../../configuration";
 import {REST_FEIL} from "../../digisos/redux/soknad/soknadTypes";
 import {NavLogEntry, NavLogLevel} from "../../digisos/redux/navlogger/navloggerTypes";
+import {ENV} from "@navikt/nav-dekoratoren-moduler/csr/functions/csr";
+
+export function detectDeploydEnvironment(): Exclude<ENV, "localhost"> {
```

Comment:
Bør kanskje renames til eks `getDecoratorEnvForDeploydEnvironment`? Det er litt herk med env-variabler i klientside-frontend, så tanken her er at vi returnerer riktig dekoratør for de ulike miljøene og defaulter til prod-dekorator.